Typos, wrong links and file added into .gitignore

TaiLinhares opened the pull request:
Hi,
I noticed some mistakes in the files and corrected them. Feel free to accept it or not.
Bests,

Tai

TaiLinhares committed:
Merge pull request #1 from pyladieshamburg/master

Add updates from Pyladies

TaiLinhares committed:
Changed the link to the initial-setup.md file

:)

TaiLinhares committed:
Changed the link to the file humidity-sensor-setup.md

:)

TaiLinhares committed:
Typo: Liniking was substitute for Linking

Not that important, but why not? :)

TaiLinhares committed:
Deleted a missing "the" on the title

Once more nothing that important, but I think it is worthy. Excuse me for not commiting the last pull request together, I haven't seen this one before.

TaiLinhares committed:
Added send-pubnub.py to private files

As suggested in 2-influxdb.md

TaiLinhares committed:
Extension name was misspelled, correct is nohup

:)

TaiLinhares committed:
Updated the path to the temperature-logging.conf file

I suppose that everybody who followed the tutorial till now will have this file in the same folder in Raspberry pi. Nohup won't find it by giving only the file name.

terezaif commented on .gitignore:
```diff
@@ -105,4 +105,5 @@ venv.bak/
-data-collecting/pubnub.ini
\ No newline at end of file
+data-collecting/pubnub.ini
+data-collecting/send-pubnub.py
```
was this one on purpose? I think it will remove the python script from the repo

TaiLinhares replied:
You're right! I thought it would be a good idea to let it in .gitignore as default, because it contains real sensitive data. One possibility would be to rename send-pubnub.py to send-pubnub-template.py. What do you think?

terezaif replied:
the private data is in the pubnub.ini config and not in the code. The pubnub.ini is in the gitignore, and we got a template that is public. So the send-pubnub.py has only code and reads from a private config file, pls check that that is the same in your setup

TaiLinhares replied:
Hi Tereza, you're right. I missunderstood what was written. You suggest to add the log file to gitignore and not the send-pubnub.py file. I'll fix this part. :)

TaiLinhares committed:
Comment out the send-pubnub.py file in .gitignore

So that the user just has to uncomment the file before submitting to github.

TaiLinhares committed:
User must uncomment line for send-pubnub.py in .gitignore

Now you just have to uncomment the file to avoid that sensitive data is uploaded to github.

TaiLinhares committed:
Undo comment about file added to gitignore

TaiLinhares committed:
Deleted send-pubnub from gitignore

TaiLinhares committed:
Copying the changes made in the pyladies version of the repository
